mscore/plugin/pluginCreator.ui

```diff
@@ -187,6 +136,25 @@
     <string>Save As…</string>
    </property>
   </action>
+  <action name="actionRun">
+   <property name="text">
+    <string>Run</string>
+   </property>
+   <property name="toolTip">
+    <string>Run plugin</string>
+   </property>
+   <property name="shortcut">
+    <string>Ctrl+R</string>
```

Comment:
Would macOS use Cmd+F? Should that be mentioned (e.g. what is the convention, is it good enough to only mention the Windows/Linux shortcut)? In places like https://musescore.org/en/handbook/keyboard-shortcuts it always points out the macOS equivalent shortcut key.

Reply:
Did you mean Cmd+R?
Isn't Cmd+R used for searching?

Reply:
Since the ui files are automatically generated, I don't know how it works exactly for cross platform.

Reply:
I guess Xcode has the universal macos shortcut for Run. Do you know what it is?

Reply:
Run didn't have a shortcut assigned, before this PT, not via ther UI file, so it should not with/after this PR

Reply:
Does that mean that i need to move the shortcut to the .cpp, or that I should completely remove it?

Reply:
remove it

Reply:
And should

> Shortcuts should never be implemented in the ui files. Use the cpp instead.

Be added to the coding rules?

Reply:
> remove it

I like having a shortcut for that, and it doesn't conflict with any other.
In fact, I don't find any reason not to have one except that it may not belong to this PR.

Reply:
I set the shortcut to F4. Does that seem Ok for you? Is there any better default? 
... Or should I still remove that shorcut?

Reply:
Well, F4 is closing Window on Windows. Anyway, I'd rather not have shortcits being hardcoded in UI files, but configurable and in shortcuts.cpp

Reply:
Its alt+F4 that closes the window?

Reply:
I put the shotcut back to the .cpp